### ISSUE TITLE
New curation type "protein localization dependency"

### ISSUE THREAD

**Issue by pombase-admin** [pombase/canto#489]
New curation type "protein localization dependency"

This is a slightly odd one. We would  like to have an annotation type "protein localization dependency"
This would be identical  to the phenotype annotation, but would  only show the terms in the phenotype ontology below "protein localization", and would have a slightly different text.
Explaining that  "localization dependencies"  are captures as phenotypes of the mutated gene, with the  protein  whose localization is being monitored added in the "optional information section"
(and later will be replaced by an annotation extension)

We think this would encourage  the community to curate 'localization dependencies" which they don't seem to think of as "classical phenotypes"

(I thought this was already on the tracker but I can't see it)

This may require
https://sourceforge.net/p/pombase/curation-tool/428/

Original comment by: ValWood

**Comment by kimrutherford** [comment]
Should be easy to implement after #1023.

**Comment by kimrutherford** [comment]
This should be straightforward now.  How urgent is this new curation type?

**Comment by ValWood** [comment]
Not super urgent. I grouped with 'new data types' which I think comes below things like 'file export' and 'filtering related tasks' in the current priorities list  (mainly because we don't curate a lot of these, and  we can do them in artemis for now...)

**Comment by ValWood** [comment]
Low priority, I think some of these  would be helpful eventually, but there might be better ways to do this. As time goes on people seem to be finding these terms more anyway, probably a they see them on pages ....

**Comment by ValWood** [comment]
This might be useful to do, even though people are now much better at making these annotations.
